python/docs/conf.py

```
@@ -54,3 +58,45 @@
 napoleon_type_aliases = None
+
+
+class NanobindEnumValueDocumenter(AttributeDocumenter):
+    """Document per-value enum docstrings exposed by nanobind."""
+
+    objtype = 'nanobind-enum-value'
+    directivetype = 'attribute'
+    priority = AttributeDocumenter.priority + 1
+
+    @classmethod
+    def can_document_member(cls, member, membername, isattr, parent):
```

Ruff (0.16.1)

[warning] Missing return type annotation for classmethod `can_document_member`

(ANN206)